.travis/linux.install.sh

```diff
@@ -30,12 +30,12 @@ sudo apt-get install --no-install-recommends \
     libxmu-dev \
     python \
     python-wxgtk3.0 \
+    python-six \
```

Comment:
Please make sure the additional action is actually taken for the standalone win installer, OSGeo4W, and Mac (installer and any brew) by testing or raising this on the ML or in the Trac issue.

Reply:
added in https://trac.osgeo.org/grass/changeset/74469